web/html/src/components/table/TableDataHandler.tsx

```diff
@@ -273,9 +280,10 @@ export class TableDataHandler extends React.Component<Props, State> {
     const fromItem = itemCount > 0 ? firstItemIndex + 1 : 0;
     const toItem = firstItemIndex + itemsPerPage <= itemCount ? firstItemIndex + itemsPerPage : itemCount;
     const isEmpty = itemCount === 0;
+    const selectEnabledItems = currItems.filter((curr) => this.props.isSelectEnabled?.(curr) ?? true);
```

Comment:
This iterates over the whole data unnecessarily if there's no selection logic bound, not a problem in most views, but in some views where the data set is large I would try and avoid this a-la

```ts
const items = typeof this.props.selectable === "undefined"
  ? currItems
  : currItems.filter(...);
```

Reply:
right, will fix